src/domain/accessibilityStatement/accessibilityStatement.module.scss

```diff
@@ -0,0 +1,19 @@
+@import 'layout';
+@import 'variables';
+
+.accessibilityStatement {
+  margin-bottom: $largeMargin;
+  @include respond-below(md) {
+    margin: 0 $baseMargin $largeMargin $baseMargin;
+  }
+
+  h1 {
+    font-size: var(--hds-text-xxl);
```

Comment:
We should not tho, even its not fitting our eyes, to keep consistency of HDS.
If h1 was too big, change it to h2 then h3 h4..

Reply:
That is also wrong, we should keep the semantics of the content of the document. Especially for h1. I'll remove these, then we can negotiate with the content creator.